script/Deploy.s.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8;
+
+import {Script, console} from "forge-std/Script.sol";
+
+import {BungeeApproveAndBridge, ISocketGateway} from "src/BungeeApproveAndBridge.sol";
+
+interface ICreateX {
+    function computeCreate3Address(bytes32 salt) external view returns (address computedAddress);
+    function deployCreate3(bytes32 salt, bytes memory initCode) external payable returns (address newContract);
+}
+
+contract DeployScript is Script {
+    BungeeApproveAndBridge public bungeeApproveAndBridge;
+
+    ISocketGateway constant SOCKET_GATEWAY = ISocketGateway(0x3a23F943181408EAC424116Af7b7790c94Cb97a5);
+
+    ICreateX constant createX = ICreateX(0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed);
```

Comment:
Interesting, I didn't know this contract: https://github.com/pcaversaccio/createx
Any advantage of doing just as instructed here?  https://getfoundry.sh/guides/deterministic-deployments-using-create2.html

Reply:
https://github.com/cowprotocol/approve-and-bridge/pull/2#discussion_r2219139867